frontend/packages/console-dynamic-plugin-sdk/src/app/components/status/PopoverStatus.scss

```diff
@@ -0,0 +1,7 @@
+.popover-status {
```

Comment:
```suggestion
.co-popover-status {
```
We prefix with `.co-`

Reply:
Further, an element should not exist without a block.  This should really be `.co-popover-status-button`.